katas/content/superdense_coding/bob_decodes_message/Placeholder.qs

```diff
@@ -0,0 +1,14 @@
+namespace Kata {
+    operation DecodeMessageFromQubits(qAlice : Qubit, qBob : Qubit) : (Bool, Bool) {
+        // Implement your solution here...
+
+        // Returning the qubit states after measuring
+        return (MResetZ(qAlice) == One, MResetZ(qBob) == One);
```

Comment:
I would do a placeholder with `return (false, false)`, and let the learner do the rest - this way we're giving away most of the solution already :-)

Reply:
Thanks for the suggestion. I updated the testing harness to do a reset of the qubits.

I was getting the below error without it, so I used reset operation to get away with that. 

```text
[ERR_ASSERTION]: Exercise "superdense_coding__bob_decodes_message" has compilation or runtime errors
        //  when using the placeholder as solution. Compilation and runtime errors:
        //   Message = runtime error: Qubit1 released while not in |0⟩ state
       ```